src/plugins/elasticsearch/index.js

```diff
@@ -1,3 +1,6 @@
+var _ = require('lodash');
+let Boom = require('boom');
```

Comment:
Yeah, that would be ideal.  I used `var` because the file was already using `var`, and I used `let` because I had a momentary brain aneurysm.

Rather than blocking this PR, mind if I just issue a follow up that converts this whole file to ES6?